src/context/AuthContext.js

```diff
@@ -57,6 +62,8 @@ export const AuthProvider = ({ children }) => {
       const response = await mascotapi.post('signingoogle', { token: tokenId });
       const responseToken = response?.data?.token?.jwtoken;
       setUser(response?.data?.user);
+      if (response.status >= 400) setAuthError(response.data.err || response.data.message); // FIXME: err -> message
```

Comment:
no entiendo el comentario :s

Reply:
en realidad el comentario es como pa mi ajsajd xD es que ya no sera err por que se cambiara a message (el estandar que usaremos)... por mientras puse los 2 pero al final solo quedara 'response.data.message'

Reply:
entonces cambiar comentarios y dejar onda "TODO: change this when backend messages has been standarized"